modules/policy/engine/src/main/java/org/eclipse/sw360/antenna/policy/engine/Rule.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+/**
+ * Interface for a generic rule with the standard fields relevant for all policy rules
+ */
+public interface Rule {
+    /**
+     * @return The business id of the rule defined by the policy responsible
+     */
+    String getId();
+
+    /**
+     * @return The business name of the rule defined by the policy responsible
+     */
+    String getName();
+
+    /**
+     * @return The description given to the user of the policy in case of a policy violation
+     */
+    String getDescription();
+
+    /**
+     * @return The configured severity of a policy violation
+     */
+    RuleSeverity getSeverity();
```

Comment:
To me it feels a bit odd to have the severity attached to the rule. I believe a "severity" should be attached to an "issue", and a "rule" reports (an) issue(s). That way issues (with severities) could also be reported / generated by other means that rules, e.g. manually entered by users.

Reply:
Ok, this I do not understand, from the systems I know, you always configure the severity with a rule, e.g., in static code analysis, we always configured the rule to have an impact. Also compiler create a severity by the violated rule? How can the polica engine define a severity if it is not based on the rule?